src/server/graphql/queries/__tests__/activeProOrgCount.test.js

```diff
@@ -24,11 +24,12 @@ test('new Pro org increments counts of Pro orgs', async () => {
   const {user} = await mockDB.init();
   const authToken = mockAuthToken(user[1], {rol: 'su'});
   // TEST
+  const initial = await activeProOrgCount.resolve(undefined, undefined, {authToken});
   await mockDB.newOrg({tier: PRO});
   const next = await activeProOrgCount.resolve(undefined, undefined, {authToken});
 
   // VERIFY
-  expect(next >= 1).toBe(true);
+  expect(next - initial).toEqual(1);
```

Comment:
is there a way to avoid this type of test? doing it this way means we have to always run tests in serial instead of parallel.